### ISSUE TITLE
bibliography: Provide access to less selective Word version

### ISSUE THREAD

**Issue by musicEnfanthen** [music-encoding/music-encoding.github.io#21]
In PR #19 @pe-ro uploaded an older, less selective Word version of the bibliography that "includes items that simply mention MEI in passing, are student work, or that, in a very small number of cases, aren't correct or flattering" and proposed that "Perhaps, however, we should provide access to this information, warts and all".

**Comment by musicEnfanthen** [comment]
@pe-ro Do you have in mind a specific location where these information / this file should be stored finally? 

Two possible options (not exclusive) :

1) Integrating the additional items in the current version (bib file). But I guess, there were some good reasons to exclude some items back then and to make the bibliography more selective, so this would contradict these reasons.

2) Adding the Word file as it is (with version date) to the `Archive` section of the MEI website where access to other documents is provided already.

**Comment by bwbohl** [comment]
Wouldn't it be nice to go without MS Word, e.g. just a txt file?

**Comment by musicEnfanthen** [comment]
Yes, of course. Or a PDF file? (Just recognized that most of the files in `Archive` are PDFs). Still remains the question how to handle this file, where to put it at all?

From your reaction I assume that archiving it under `Archive` is an option open to consensus?

**Comment by bwbohl** [comment]
It's been hell of long time ago when I first created a music encoding initiative Zotero Group an it's been somewhat orphaned since then…
but why not use it for that purpose?
https://www.zotero.org/groups/1446386/music-encoding-initative

**Comment by bwbohl** [comment]
forget it… you've already got a bibs page and I got the nature of the word file wrong in the first place…

on the other hand with something in Zotero (maybe also bib base) we could use tags to mark those passer by / student work literature things an extract them to a separate list

**Comment by bwbohl** [comment]
how was the bib base generated? from a bibtex file? where and how is it maintained?

**Comment by musicEnfanthen** [comment]
Initially, I used Citavi to generate the bib file what was much faster than adding all the items by hand (but it has its own drawbacks, too). 

Now the bib file is in `resources/mei_bibliography.bib` and single items can be added directly in the file.

Indeed, BibBase supports managing the bibliography via Zotero, but there are some issues: https://bibbase.userecho.com/communities/1-general#module_20/search/Zotero
(But maybe you could try to import the bib file into your Zotero Group and see what´s possible?)

It is also possible to add `keywords` directly in the bibfile, and filter the bibbase list by keywords (if I understand your idea with the "tags" correctly).

**Comment by rettinghaus** [comment]
I uploaded our current bibliography to Zotero, so maybe we could add those additional publications there?

**Comment by bwbohl** [comment]
the Zotero Library is available at https://www.zotero.org/groups/1446386/music-encoding-initative
